static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -91,8 +91,8 @@ export function EventFeatureFlagList({
     event,
   });
 
-  const hasFlagContext = !!event.contexts.flags;
-  const hasFlags = Boolean(hasFlagContext && event?.contexts?.flags?.values.length);
+  const hasFlagContext = !!event.contexts?.flags?.values;
```

Comment:
could also go over here and add a `?` too: 
https://github.com/getsentry/sentry/blob/8c07f13b1909fc3191d10f24c853c970815055cf/static/app/types/event.tsx#L646